src/ElementBlock.js

```diff
@@ -0,0 +1,48 @@
+import ElementPropTypes from './propTypes';
+
+export const configSchema = {
+    isFullWidth: {
+        label: `Full width`,
+        type: ElementPropTypes.bool
+    },
+    hasHorizontalPadding: {
+        label: `Horizontal spacing`,
+        type: ElementPropTypes.bool
+    },
+    topPadding: {
+        label: `Top spacing`,
+        type: ElementPropTypes.oneOf(['None', 'Small', 'Medium', 'Large'])
+    },
+    bottomPadding: {
+        label: `Top spacing`,
+        type: ElementPropTypes.oneOf(['None', 'Small', 'Medium', 'Large'])
+    },
+    showOnDesktop: {
+        label: `Show on desktop`,
+        type: ElementPropTypes.bool
+    },
+    showOnTablet: {
+        label: `Show on tablet`,
+        type: ElementPropTypes.bool
+    },
+    showOnMobile: {
+        label: `Show on mobile`,
+        type: ElementPropTypes.bool
+    },
+    customAttrs: {
+        label: `Custom element attributes`,
+        type: ElementPropTypes.arrayOf(
+            ElementPropTypes.shape({
+                name: {
+                    label: `Name`,
+                    tooltip: `All entered values are automatically prefixed with "data-element-". Example: "data-element-your-custom-attribute-name"`,
+                    type: ElementPropTypes.string
+                },
+                value: {
+                    label: `Value`,
+                    type: ElementPropTypes.string
+                }
+            })
+        )
+    }
+};
```

Comment:
should we also have some tests against the legacy configs, which didn't have `label` and `type`, but instead used `fromCamelToSentence` to convert the key into a label?